mne/utils/dataframe.py

```diff
@@ -51,6 +51,7 @@ def _convert_times(times, time_format, meas_date=None, first_time=0):
 
 
 def _inplace(df, method, **kwargs):
+    # TODO VERSION can be removed once pandas>=1.5 is required
```

Comment:
confused; pandas 2.1 is the lower pin in pyproject.toml and environment.yaml. I think that means we can drop the `else` branch of hte conditional (the one with `inplace`) and mark this as

```suggestion
    # TODO VERSION can be removed once pandas>=3.0 is required
```